crawler/dockercontainer.py

```diff
@@ -24,6 +27,7 @@
 
 logger = logging.getLogger('crawlutils')
 
+HOST_PID_BASEDIR = '/var/run/'
```

Comment:
oh..forgot to remove it. Earlier i though of keeping the pids of any processes started by crawler in a file, so that we they can be managed separately. But, later decided against it. 
Will remove it.